cylc/flow/scripts/check_versions.py

```diff
@@ -88,7 +88,10 @@ def main(_, options, *args):
     versions = {}
     for platform_name in sorted(platforms):
         platform = get_platform(platform_name)
-        cmd = construct_ssh_cmd(['version'], platform)
+        host = get_host_from_platform(
+            platform, badhosts=None
+        )
+        cmd = construct_ssh_cmd(['version'], platform, host)
```

Comment:
Inconsistent styling here, for `host = ...` and `cmd = ...`.

I'd leave it all on one line, or go all the way for readability like this:
```suggestion
        host = get_host_from_platform(
            platform,
            badhosts=None
        )
        cmd = construct_ssh_cmd(
            ['version'],
            platform,
            host
        )
```